consul/flood.go

```diff
@@ -46,7 +46,6 @@ func (s *Server) Flood(portFn servers.FloodPortFn, global *serf.Serf) {
 	}()
 
 	for {
-	WAIT:
```

Comment:
This particular change scares me. Vet said it was unreachable code, which I see, but I also worry it's wrong.